Makefile

```diff
@@ -16,134 +17,93 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #*******************************************************************************
-
 ifeq ($(BOLOS_SDK),)
-$(error BOLOS_SDK is not set)
+$(error Environment variable BOLOS_SDK is not set)
 endif
 
-dummy_submodules := $(shell git submodule update --init --recursive)
+include $(BOLOS_SDK)/Makefile.defines
 
-ifeq ($(TARGET_NAME),TARGET_NANOS)
-APP_STACK_SIZE:=880
```

Comment:
Are you sure this can be removed?
How to really validate this?

Reply:
Stack size in SDK's Nano S linkerscript is `1024` so more than what was defined here. Not sure why this app would need a custom size... My guess is that it is an obsolete definition that was present in early apps makefiles.